### ISSUE TITLE
springfox doesn't work with Jackson 2.9.0.pr4

### ISSUE THREAD

**Issue by beihaifeiwu** [springfox/springfox#1909]
When I upgrade my project to Spring Boot 2.0.0.M2, I found that springfox cannot show the model !!!
After I dig into it I got the following problem: 

* **springfox.documentation.schema.property.BeanPropertyDefinitions**
```java
  private static String getName(
      PropertyNamingStrategy naming,
      BeanPropertyDefinition beanProperty,
      MapperConfig<?> config) {

    PojoPropertyBuilderFactory factory = new PojoPropertyBuilderFactory();
    POJOPropertyBuilder prop = factory.create(config, beanProperty);
    return naming.nameForField(config, prop.getField(), beanProperty.getName());
  }
```
The **prop.getField()** just return null, why is that ?

**Comment by dilipkrish** [comment]
This is related to #1773

**Comment by dilipkrish** [comment]
This is a related to #1773. Spring 5 and spring-boot 2.0.0 and consequently jackson 2.9.x (which is not supported)

**Comment by dilipkrish** [comment]
This is now fixed with support for jackson 2.9.5 via #1899
